history/report_utils.py

```diff
@@ -589,7 +802,203 @@ def compare_calcs(base, new, name, template_args, plot_paths):
         aggs["Tax Liability"].append(new_aggs["combined"])
         aggs["Tax"].append("New Combined")
         aggs["Year"].append(year)
+
+        aggs2["Value"].append(cur_salary_wage)
```

Comment:
Could put the variables and strings in lists and reduce the lines of code by using a loop or Python iterable method.  I think doing so would was the ability to follow what's happening and reduce errors.

Reply:
have made the adjustment in this [ new commit](https://github.com/PSLmodels/taxdata/pull/417/commits/109cac61006c497417401e2d67cd31b922218cdc)